src/smoke_tests/Makefile

```diff
@@ -0,0 +1,22 @@
+
+EXECUTABLES = simple threads
+
+CC ?= gcc
+
+PAPIROOT ?= /path/to/papi/install/prefix
+CFLAGS ?= -O0 -pthread -I$(PAPIROOT)/include
```

Comment:
Applications will likely use -O2

Reply:
This was provided by Heike.  I don't have a preference.